### PR TITLE
Publish package to NPM

### PR DESCRIPTION
**Pull request by wilwade**
## Done
- Update action versions
- Update node versions
- NPM Publish workflows

## Results
- Working Publish RC run: https://github.com/AmplicaLabs/siwf/actions/runs/8512290132
- NPM Package: https://www.npmjs.com/package/@amplica-labs/siwf

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -26,7 +26,7 @@ jobs:
           node-version: ${{ matrix.node-version }}
 
       - name: Install pnpm
-        uses: pnpm/action-setup@v2
+        uses: pnpm/action-setup@v3
```
Comment (wilwade) [inline .github/workflows/ci.yml:29]: Bumped for node20 on all these action version bumps

File: .github/workflows/deploy.yml [hunk 2]
```diff
@@ -64,4 +64,59 @@ jobs:
     steps:
       - name: Deploy
         id: deployment
-        uses: actions/deploy-pages@v1
+        uses: actions/deploy-pages@v4
+
+  publish-npm-rc:
```
Comment (wilwade) [inline .github/workflows/deploy.yml:69]: Each merge we publish a version with the `@next` and the `0.0.0-sha`

File: .github/workflows/release.yml [hunk 3]
```diff
@@ -0,0 +1,83 @@
+# yaml-language-server: $schema=https://json.schemastore.org/github-workflow.json
+
+name: Release
+run-name: Cut Release ${{github.event.inputs.release-version || github.ref_name}}
+concurrency:
+  group: ${{github.workflow}}-${{github.ref}}
+  cancel-in-progress: false
+on:
+  push:
+    tags:
```
Comment (wilwade) [inline .github/workflows/release.yml:10]: The versioned release from tags

File: .github/workflows/deploy.yml (hunk 2 above)
Comment (wilwade) [reply .github/workflows/deploy.yml:69]: To not do it. NPM really doesn't like removing old versions

**Review by JoeCap08055 — APPROVED** [review]

**Pull request merged**